lib/http.js

```diff
@@ -47,6 +58,15 @@ function httpRequest(sdk, options) {
         cookies.setCookie(config.STATE_TOKEN_COOKIE_NAME, res.stateToken, res.expiresAt);
       }
 
+      if (res && options.responseCacheDuration) {
+        var cache = httpCache.getStorage();
```

Comment:
It might be nice to provide a method for "updateStorage" - namely, underneath the hood it's going to get, write, and set, but we can just set a new url with the value here.